django/channels/channel.py

```diff
@@ -0,0 +1,71 @@
+from __future__ import unicode_literals
+
+from django.channels import DEFAULT_CHANNEL_LAYER, channel_layers
+from django.utils import six
+
+
```

Comment:
You might want `@python_2_unicode_compatible` on some of these classes that have `__str__()`.